cmd/list.go

```diff
@@ -26,15 +26,80 @@ import (
 	"fmt"
 	pipeline2 "github.com/Azure/azure-pipeline-go/pipeline"
 	"strconv"
+	"strings"
 
 	"github.com/spf13/cobra"
 
 	"github.com/Azure/azure-storage-azcopy/common"
 	"github.com/Azure/azure-storage-azcopy/ste"
 )
 
+type rawListCmdArgs struct {
+	// obtained from argument
+	sourcePath string
+
+	Properties      string
+	MachineReadable bool
+	RunningTally    bool
+	MegaUnits       bool
+}
+
+var validProperties = []string{
+	"LastModifiedTime", "VersionId", "BlobType", "BlobAccessTier", "ContentType", "ContentEncoding",
```

Comment:
>LastModifiedTime [](start = 2, length = 16)

Sorry I should have given you this info earlier, I found the original ask and it included lease status too.

Reply:
No problem. Changes weren't that big. Added `LeaseStatus`, `LeaseState`, and `LeaseDuration` in the `storedObject` as well.